deephyper/contrib/layers/stack_mpnn.py

```diff
@@ -1,7 +1,99 @@
 import tensorflow as tf
+import numpy as np
+import tensorflow.keras.backend as K
+from numpy.random import seed
 from tensorflow.keras import activations
 from tensorflow.keras.layers import Dense
 
+seed(2020)
+tf.random.set_seed(2020)
```

Comment:
The random seed should never be set globally otherwise it has side-effect. If we import any object from this module the random state will be fixed. For numpy, it is better to use local `np.random.RandomState`. The seed to fix a neural network training is performed in the `run_base_trainer`: https://github.com/deephyper/deephyper/blob/develop/deephyper/nas/run/_run_base_trainer.py#L62

Reply:
I have removed the random seeds.